vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher.go

```diff
@@ -276,11 +280,14 @@ func (f *controllerFetcher) isWellKnownOrScalable(key *ControllerKeyWithAPIVersi
 }
 
 func (f *controllerFetcher) getOwnerForScaleResource(groupKind schema.GroupKind, namespace, name string) (*ControllerKeyWithAPIVersion, error) {
+	if wellKnownController(groupKind.Kind) == node {
+		// Node is not a proper owner
```

Comment:
Can you add more context? I.e. that this is actually a scalability issue in bigger clusters. Also is it not a problem that we return no error but parent is nil? Shouldn't this be an error, since we failed to find a proper controller?

Reply:
Done.